ui/components/app/alert-system/confirm-alert-modal/confirm-alert-modal.tsx

```diff
@@ -40,16 +41,20 @@ function ConfirmButtons({
   onSubmit,
   isConfirmed,
 }: {
-  onCancel: () => void;
+  onCancel: ({ location }: { location: MetaMetricsEventLocation }) => void;
   onSubmit: () => void;
   isConfirmed: boolean;
 }) {
   const t = useI18nContext();
+  const onAlertCancel = useCallback(() => {
+    onCancel({ location: MetaMetricsEventLocation.AlertFrictionModal });
+  }, []);
```

Comment:
I think best practice is to include any properties as dependencies, so `onCancel` in this case?

Reply:
done